public/img/icons.svg

```diff
@@ -17,6 +17,9 @@
   <symbol fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 14" id="icon-checkmark">
     <path d="M2.345 7.406l3.381 3.361 6.205-6.185-.776-.775-5.429 5.409L3.101 6.63l-.756.775z" fill="currentColor"></path>
   </symbol>
+  <symbol fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 14" id="icon-test">
+    <path d="M2.345 7.406l3.381 3.361 6.205-6.185-.776-.775-5.429 5.409L3.101 6.63l-.756.775z" fill="currentColor"></path>
+  </symbol>
```

Comment:
This is currently the same as the checkmark, but I think it's likely to evolve to its own icon.